src/js/views/patients/sidebar/engagement/engagement-content.hbs

```diff
@@ -0,0 +1,22 @@
+<div class="engagement-sidebar__heading">{{ @intl.patients.sidebar.engagement.engagementContentTemplate.statusHeading }}</div>
+<div class="engagement-sidebar__item">
+  {{#if engagement.status}}
+    <span class="engagement-status__icon {{ engagement.status }} u-margin--r-4">{{fas "circle"}}</span>{{formatMessage (intlGet "patients.sidebar.engagement.engagementContentTemplate.status") status=engagement.status}}
+  {{else}}
+    <span class="engagement-sidebar__no-engagement">{{ @intl.patients.sidebar.engagement.engagementContentTemplate.notAvailable }}</span>
```

Comment:
This could be handled in the `status` `select`

Reply:
This is identical to the logic in the patient sidebar view. So we'll probably want to change it in both places.